src/Core/Services/MetadataProviders/SqlMetadataProvider.cs

```diff
@@ -300,10 +298,8 @@ public bool TryGetEntityToColumnMappings(string entityName, [NotNullWhen(true)]
             return false;
         }
 
-        /// <summary>
-        /// Given entity name, gets the column to entity mappings if present.
-        /// </summary>
-        public bool TryGetColumnToEntityMappings(string entityName, [NotNullWhen(true)] out IReadOnlyDictionary<string, string>? mappings)
+        /// <inheritdoc/>
+        public bool TryGetFieldToEntityMappings(string entityName, [NotNullWhen(true)] out IReadOnlyDictionary<string, string>? mappings)
         {
             Dictionary<string, string>? columntoEntityMappings;
```

Comment:
please fix the name of this variable to `columnToExposedNameMap` for e.g.

Reply:
`TryGetBackingFieldToExposedFieldMap` to align with usage of `field`. Just so there is no confusion since we don't need to use the term `column` for an interface applying to all DBs such as Cosmos.

Reply:
Fixed this to new name.